install/uninstall.sh

```
@@ -1,6 +1,13 @@
 #!/bin/bash
 
 clear
+if [ $EUID != 0 ]; then
+    echo "This script requires root however you are currently running under another user."
+    echo "Please use 'sudo su' to switch to root user before running this script."
+    echo "卸载脚本需要使用 root 权限，请先使用 sudo su 切换到 root 用户后再运行此脚本。"
+    # sudo "$0" "$@"
+    exit $?
```

Shellcheck (0.11.0)

[warning] This $? refers to echo/printf, not a previous command. Assign to variable to avoid it being overwritten.

(SC2320)